open_spiel/python/algorithms/expected_game_score.py

```diff
@@ -47,10 +52,22 @@ def policy_value(state, policies):
     for action, action_prob in state.chance_outcomes():
       child = state.child(action)
       values += action_prob * policy_value(child, policies)
-    return values
   elif state.is_simultaneous_node():
-    raise NotImplementedError(
-        "Policy Value is not implemented for simultaneous games")
+    values = np.zeros(shape=num_players)
+    action_prob_dicts = []
+    for player in range(num_players):
+      action_prob_dicts.append(
+        policies.action_probabilities(state, player))
+    actions = itertools.product(*action_prob_dicts)
+    probabilities = itertools.product(
+      *[action_prob.values() for action_prob in action_prob_dicts])
+    for action_tuple, probability_tuple in zip(actions, probabilities):
```

Comment:
It would be clearer to add a helper function that produced a list of [(actions), prob] or something like it for a policy in a simultaneous move game. That would make this code simpler, and also the helper function could be independently tested.

Reply:
Should I implement the function in C++ and add a python wrapper?
I agree that this would be easier.

Reply:
@elkhrt ?

Reply:
Python is fine unless you have C++ use cases in mind.

Reply:
If you don't need it from C++, then pure Python is fine too - whichever is easier.

Reply:
Done in `policy.py`. Thanks!